test/google_scraping/dashboard/keywords_test.exs

```diff
@@ -26,6 +26,54 @@ defmodule GoogleScraping.Dashboard.KeywordsTest do
     end
   end
 
+  describe "apply_filters_to_user_keywords/2" do
+    test "with provided url_contains filter, returns count of urls" do
```

Comment:
I think you tend to use `given _` in tests right? 
```suggestion
    test "given provided url_contains filter, returns count of urls" do
```

Reply:
Fixed here: https://github.com/Nihisil/nimble-certification-elixir/pull/52/commits/9bf3e9bcefa159e5ec33ae5e9cc5d54e1a9acb7e